db/migrate/20200721224611_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :google_token
+      t.string :email
+      t.string :photo, default: 'https://ftnj.com/wp-content/uploads/2018/09/female-headshot-silhouette.jpg'
```

Comment:
default photo is a female silhouette, I think it looks a bit gender nuetral. I'd love to have that hair
[](https://ftnj.com/wp-content/uploads/2018/09/female-headshot-silhouette.jpg)

Reply:
ooh, nice find on this one. looks a bit more 'modern' than the standard myspace default

Reply:
I like this default photo. That hair looks great :)